.github/workflows/test.yml

```diff
@@ -13,4 +13,5 @@ jobs:
         go-version: 1.13.x
     - uses: actions/checkout@v2
     - run: go get -v -t -d ./...
-    - run: go test -v ./...
+    - run: go test -v parse_test.go
```

Comment:
`parse_test.go` and `test_paths_test.go` aren't added in this PR, is this change related?

It does seem like some tests for the new code would be useful however, so maybe add a `test_main.go`?

Reply:
Yea I will add test_main.go. This change is needed since we don't have to run `test_main.go` for every PR